compiler/circle-execution-plan/src/ExecutionPlanner.test.cpp

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <loco.h>
+#include "ExecutionPlanner.h"
+
+#include <luci/IR/CircleNodes.h>
+
+#include <gtest/gtest.h>
+
+TEST(ExecutionPlannerTest, op_with_multiple_outputs)
+{
+  auto g = loco::make_graph();
+
+  // Create nodes
+  auto input = g->nodes()->create<luci::CircleInput>();
+  auto split = g->nodes()->create<luci::CircleSplit>();
+  auto split_out1 = g->nodes()->create<luci::CircleSplitOut>();
+  auto split_out2 = g->nodes()->create<luci::CircleSplitOut>();
+  auto split_dim = g->nodes()->create<luci::CircleConst>();
+
+  // Build up initial graph
+  auto graph_input1 = g->inputs()->create();
+  graph_input1->shape({1, 6});
+
+  input->index(graph_input1->index());
+  input->shape({1, 6});
+  input->dtype(loco::DataType::S32);
+  input->shape_status(luci::ShapeStatus::VALID);
+  input->name("input");
+
+  split_dim->dtype(loco::DataType::S32);
+  split_dim->size<loco::DataType::S32>(1);
+  split_dim->shape({1});
+  split_dim->at<loco::DataType::S32>(0) = 1;
+  split_dim->shape_status(luci::ShapeStatus::VALID);
+  split_dim->name("split_dim_const");
+
+  split->split_dim(split_dim);
+  split->dtype(input->dtype());
+  split->input(input);
+  split->num_split(2);
+  split->name("split");
+
+  split_out1->input(split);
+  split_out1->index(0);
+  split_out1->shape({1, 3});
+  split_out1->dtype(split->dtype());
+  split_out1->name("split_out1");
+
+  split_out2->input(split);
+  split_out2->index(1);
+  split_out2->shape({1, 3});
+  split_out2->dtype(split->dtype());
+  split_out2->name("split_out2");
+
+  auto add = g->nodes()->create<luci::CircleAdd>();
+  add->name("add");
+  add->x(split_out1);
+  add->y(split_out2);
+  add->dtype(split_out1->dtype());
+  add->shape({1, 3});
```

Comment:
Note: if you are going to make larger test graphs, maybe it is easier to set only input shaped and dtypes, then run type and shape inference.